lib/test_queue/runner.rb

```diff
@@ -311,13 +338,17 @@ def enqueue_discovered_suite(suite_name, path)
 
       if @original_queue.include?([suite_name, path])
         # This suite was already added to the queue some other way.
+        # We still want to track it in the list of discovered suites though.
+        @discovered_suites << [suite_name, path]
```

Comment:
I'd move this up to before the `if` and get rid of the comment. That will get rid of the duplication of this line after the block.